{
  "schema_version": "1.4.0",
  "id": "GHSA-cwg2-qmg3-3f6x",
  "modified": "2024-10-29T18:30:37Z",
  "published": "2024-10-29T18:30:36Z",
  "aliases": [
    "CVE-2019-25219"
  ],
  "details": "Asio C++ Library before 1.13.0 lacks a fallback error code in the case of SSL_ERROR_SYSCALL with no associated error information from the SSL library being used.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-25219"
    },
    {
      "type": "WEB",
      "url": "https://github.com/chriskohlhoff/asio/commit/93337cba7b013150f5aa6194393e1d94be2853ec"
    },
    {
      "type": "WEB",
      "url": "https://github.com/chriskohlhoff/asio/compare/asio-1-12-2...asio-1-13-0"
    },
    {
      "type": "WEB",
      "url": "https://think-async.com/Asio"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": null,
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2024-10-29T17:15:03Z"
  }
}